src/sagemaker_xgboost_container/algorithm_mode/serve.py

```diff
@@ -21,11 +22,11 @@
 import flask
 import gunicorn.app.base
 
-from sagemaker_xgboost_container.algorithm_mode import integration, serve_utils
+from sagemaker_xgboost_container.algorithm_mode import integration
+from sagemaker_xgboost_container.algorithm_mode import serve_utils
 from sagemaker_xgboost_container.constants import sm_env_constants
 
 
-SAGEMAKER_BATCH = os.getenv("SAGEMAKER_BATCH")
```

Comment:
Why did you remove this? This was previously parsed once during env initialization, but now this gets parsed every inference call. Add back.